discover/deployment/articles-dxp/08-liferay-enterprise-search/09-xpack-monitoring.markdown

```diff
@@ -42,9 +39,6 @@ for details.
 
         ./bin/kibana-plugin install x-pack
 
-    If you're not using X-Pack Security on the Elasticsearch cluster, skip this
-    step.
-
```

Comment:
@JosemariaMunoz 
Do you need to install x-pack on Kibana if you'r not using X-Pack Security? I can ask the Search team if there's any doubt here.

Reply:
Yes @rbohl, a customer tell me that they were trying to install just Monitoring and waste so much time reading the documentation because it is needed.
I make a test and I had to install xpack too, I think is because you need to set xpack.security.enabled=false in kybana.yml (By default is false but xpack has to be installed is order to use it).
Anyway, please ask to Search Team in order to clarify this.
Thanks a lot!

Reply:
Thanks Jose, I did discuss with the Search team and we were convinced this isn't necessary. Let's call them in here for discussion so we can figure it out for sure. @arboliveira @xbrianlee @lipusz. Do you guys think we should keep this caveat in the docs (_if you're not using X-Pack Security, you don't need to install X-Pack onto Kibana_)? 

When I tested these docs steps I'm pretty sure I had already configured X-Pack security on the ES server, so I haven't tested this myself.

Reply:
Perfect @rbohl, maybe they know any additional step to install 'Monitoring' without xpack, in which case we can document this step too, but my experience was that I had to install it.
Thanks!!

Reply:
Sorry @rbohl, I misinterpreted your question on Friday. X-Pack is unnecessary if both X-Pack Monitoring and Security aren't needed. However if only Monitoring is needed, you'll need to set  `xpack.security.enabled=false` in elasticsearch.yml ([propagates to all Kibana instances that are linked to the instance](xpack.security.enabled)) because Security is enabled by default.

Reply:
No problem, glad we got to the bottom of it. I have no idea where that if statement came from then, but I'm glad we've finally killed it.